### ISSUE TITLE
Add/New-AzLoadBalancerFrontendIpConfig Public IP Prefix Support

### ISSUE THREAD

**Issue by MrCaedes** [Azure/azure-powershell#12214]
## Description of the new feature
Public IP Prefixes are GA, however, do not appear to be supported when attempting to utilise via `Add-AzLoadBalancerFrontendIpConfig` and `New-AzLoadBalancerFrontendIpConfig` when setting up (or modifying) a Load Balancer. 

This is contrary to the capabilities provided via the Azure Portal, which allows FEC to be setup against the PIPP object - which can then be utilised by Outbound rules without an issue, when manually setup. 

![Portal Example](https://i.imgur.com/2awsA3e.png)

Ideally, the aforementioned cmdlets should be updated to allow both `Microsoft.Azure.Commands.Network.Models.PSPublicIpPrefix` and `Microsoft.Azure.Commands.Network.Models.PSPublicIpAddress` to be utilised for the `-PublicIPAddress` flag to bring in-line with the deployments available via the portal (and ARM).

## Current Workaround
The current workaround that I've had to resort to is completely cutting out the use of the aforementioned cmdlets, and fudging together the below within my workflow:

```powershell
$azContext = Get-AzContext
$azProfile = [Microsoft.Azure.Commands.Common.Authentication.Abstractions.AzureRmProfileProvider]::Instance.Profile
$profileClient = New-Object -TypeName Microsoft.Azure.Commands.ResourceManager.Common.RMProfileClient -ArgumentList ($azProfile)
$tokenHeader = @{
    'Content-Type' = 'application/json';
    'Authorization' = 'Bearer ' + ($profileClient.AcquireAccessToken($azContext.Subscription.TenantId)).AccessToken;   
}

$lbConfigEP = 'https://management.azure.com/subscriptions/EXAMPLE/resourceGroups/EXAMPLE/providers/Microsoft.Network/loadBalancers/exampleLb?api-version=2020-05-01'
$currentConfig = Invoke-RestMethod -Method GET -Uri $lbConfigEP -Headers $tokenHeader 
$currentConfig.properties.frontendIPConfigurations += @{
    "name" = "portalExample2";
    "properties" = @{
        "privateIPAllocationMethod" = "Dynamic";
        "publicIPPrefix" = @{
            "id" = "/subscriptions/EXAMPLE/resourceGroups/PPIP_Container/providers/Microsoft.Network/publicIPPrefixes/exampleTest"
        }
    }
}

$updatedConfig = Invoke-RestMethod -Method PUT -Uri $lbConfigEP -Headers $tokenHeader -Body ($currentConfig | ConvertTo-JSON -Depth 100)
```

**Comment by dingmeng-xue** [comment]
Network team, please help to look into this issue.

**Comment by irenehua** [comment]
Thanks for raising the issue! This issue is being triaged. We'll get back once we have an update.

**Comment by SatishBoddu-MSFT** [comment]
Hello @MrCaedes Apologies for the delayed response. I see that this issue is opened a long time ago and no further activity had taken place. So wanted to check if you are still looking for assistance on this query? Please let us know if we are good to close this issue? @brianlehr

**Comment by MrCaedes** [comment]
@SatishBoddu-MSFT -
This appears to now be implemented, going by the [Az PowerShell 6.5.0](https://docs.microsoft.com/en-us/powershell/module/az.network/new-azloadbalancerfrontendipconfig?view=azps-6.5.0#:~:text=-PublicIpAddressPrefix%20%3CPSPublicIpPrefix%3E) reference. Going by the history on the documentation, this appears to have been added on the [13th November](https://github.com/Azure/azure-powershell/pull/13499).

Closing off as this has now been implemented / solved. 

Thank you.

**Comment by brianlehr** [comment]
Yes - this was implemented.  @MrCaedes - apologies that this issue wasn't updated appropriately when the change was made.
